config/application.rb

```diff
@@ -0,0 +1,32 @@
+require_relative 'boot'
+
+require "rails"
+# Pick the frameworks you want:
+require "active_model/railtie"
+require "active_job/railtie"
+require "active_record/railtie"
+require "action_controller/railtie"
+require "action_mailer/railtie"
+require "action_view/railtie"
+# require "sprockets/railtie"
+require "rails/test_unit/railtie"
+
```

Comment:
for the above, can we comment out all the stuff we don't want?  This is all we have for preservation_catalog:

```
require 'rails'
require 'active_record/railtie'
require 'action_controller/railtie'
```

loading all these things is one of the ways spinning up the app takes so long.  I am even wondering if we have to include "rails" as a stand alone require.